app/templates/components/billing/information.hbs

```diff
@@ -112,29 +112,62 @@
             class='billing-country'
             @showRequiredMark={{true}}
             @showValidationStatusIcons={{false}}
-            @onChange={{action (mut this.newSubscription.billingInfo.country)}} 
+            @onChange={{action (mut this.newSubscription.billingInfo.country)}}
             as |Field|
           >
             <Field.select
               @placeholder='Select country'
-              @options={{countries}} as |country|
+              @options={{this.countries}} as |country|
             >
               {{country}}
             </Field.select>
           </form.field>
       </div>
       </div>
-      <div class='form-elem-half-width inline-form-elem billing-optional-field'>
-        <form.field
-          @label='Vat Id'
-          @value={{newSubscription.billingInfo.vatId}}
-          @showValidationStatusIcons={{false}}
-          @onChange={{action (mut newSubscription.billingInfo.vatId)}} 
-          as |Field|
-        >
-          <Field.input @name='vat'/>
-        </form.field>
-      </div>
+      {{#if this.showNonZeroVatConfirmation}}
+        <div class='flex justify-between items-center mb-16'>
+          <div class='w-4/6 pr-2'>
+            Is your company registered locally for VAT/GST?
```

Comment:
@artur can we please sending the option selected by the customer for this question in the payload as well because in the billing service we need to place some validations based on what customer selected.

Reply:
@shairyar yes, that's exactly what I was going to ask you about :) Could you please give me a little bit of context with regards to how/where this flag should be sent? Thanks!

Reply:
You can send this as part of a subscription payload for example when someone submits the form when creating a subscription.

Reply:
Probably we can add a new attribute to existing [billing-info](https://github.com/travis-ci/travis-web/blob/master/app/models/billing-info.js) model and call it something like `has-local-registration: boolean`. How does it sound to you?

Reply:
Hey @shairyar, I have just added the discussed attribute to the billing info (it should be sent over with subscription data). Please take a look if it works.

Reply:
Thank you, I will have a look at this today and get back to you.